src/services/postService.js

```diff
@@ -53,9 +79,33 @@ async function checkLike(like, postID, userID){
     return postData;
 }
 
+const deletePost = async (postId) => {
+    await getPostById(postId);
+
+    const deleteResult = await postDAO.deletePost(postId);
+    throwIfError(deleteResult);
+}
+
+const deleteReply = async (postId, replyId) => {
+    const repliesOfPost = await getRepliesOfPost(postId);
```

Comment:
In my opinion, the get replies function is not needed because it can be done in one line. Creating a function adds unnecessary overhead and adds more things to track. You don't have to change it if you don't want to as this is just my opinion.
Can use 
``` Javascript
const {replies} = await getPostById(postId);
// Or
const post = await getPostById(postId);
const repliesOfPost = post.replies;
```